### ISSUE TITLE
Unable to access other team member workspaces

### ISSUE THREAD

**Issue by Vilsol** [codenvy/codenvy#2260]
I am unable to open another team members workspace, whilst I have been given all possible permissions in the team settings. I am even able to edit any configuration of the workspace and even as far as delete it, change its owner, etc.

When I open the workspace, I am greeted with the following message:
![image](https://user-images.githubusercontent.com/1759390/27019617-ca91f0ce-4f42-11e7-857b-dca2cf53c893.png)

**Reproduction Steps:**

1. Another team member signs up and creates a team
2. You sign up to that team
3. Other team member gives you all permissions
4. Third team member creates a workspace

**Codenvy version:** 5.11.2

**Comment by ashumilova** [comment]
@Vilsol the user who gets that error is team admin, right? does he have direct access to workspace (Share page on workspace details)?

Team admin can manage workspace - delete, change it's config, but not "use" (direct access to machines). By default only workspace creator can do that or he can share workspace with other team members. 

What can be fixed on our side is the UX behavior in case user is trying to access workspace with no rights.

**Comment by Vilsol** [comment]
Yes, that definitely should be changed, also, I was able to start up the workspace if it hadn't been setup yet, but as soon as it finished loading up, it threw the same error. It should not allow you to start the workspace, and present a message or even hide the workspace from the sidebar, as well as have a grayed out "Open" in the workspace page with an explication of why you can't access this workspace.

However, I think that if you are specified as a team administrator, you should be able to access any workspace under the team domain.
